cypress/integration/e2e/header-control-menu/return-to-now-button.spec.ts

```diff
@@ -0,0 +1,19 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+describe("'Come back to now' button", () => {
+  context("when schedule is loaded", () => {
+    before(() => {
+      cy.loadScheduleToMonth();
+      cy.get("[data-cy=switch-next-month]").click();
+    });
+
+    context("when month is switched to next", () => {
+      it("should get you back to current month after clicking 'return to now' button", () => {
+        cy.get("[data-cy=return-to-now]").click();
+        cy.get("[data-cy=month-name]").contains("Listopad 2020");
```

Comment:
Myślę, że lepiej by było uzależnić to od tych wartości
![image](https://user-images.githubusercontent.com/21079319/117082097-a2c17900-ad41-11eb-9455-d00d82b176d5.png)

Można to zrobić w mniej więcej taki sposób
```js
const month = StringHelper.capitilize(TranslationHelper.polishMonths[TEST_SCHEDULE_MONTH]);
const monthName  = `month ${TEST_SCHEDULE_YEAR}`;
cy.get("[data-cy=month-name]").contains(monthName);
```
![image](https://user-images.githubusercontent.com/21079319/117082158-ca184600-ad41-11eb-9e22-fb510af935d4.png)

Reply:
Actually I think that it is better to either hard code this string alongside `TEST_SCHEDULE_MONTH` and `TEST_SCHEDULE_YEAR` instead of creating new dependencies for this test.

Reply:
Według mnie trochę słabe to, że nie widać dlaczego jest listopad.

Ale ten problem można też rozwiązać w taki sposób, że dodać do `before` ustawienie czasu aplikacji na `Listopad`. (podobnie do tego jak to jest robione w funkcji `loadScheduleToMonth`).

![obraz](https://user-images.githubusercontent.com/21079319/117182226-60964700-add6-11eb-8c0d-7fcb36435611.png)

I to rozwiązanie było sensowne też pod tym względem, że test testująca powrót do teraz sam ustawia to, kiedy jest teraz.